components/supervisor/pkg/terminal/terminal_test.go

```diff
@@ -313,6 +315,43 @@ func TestConcurrent(t *testing.T) {
 	}
 }
 
+func TestClose(t *testing.T) {
```

Comment:
This test fails in our infrastructure, but runs ok in my preview env. No idea yet, why.